src/Sulu/Bundle/ContentBundle/Command/WebspaceCopyCommand.php

```diff
@@ -535,6 +543,44 @@ protected function updateSmartContentStructure(
         }
     }
 
+    /**
+     * Updates references in structure for content type `teaser_selection`.
+     *
+     * @param array $structureArray
+     * @param PropertyMetadata $property
+     * @param string $localeSource
+     * @param string $localeDestination
+     */
+    protected function updateTeaserSelection(
+        array &$structureArray,
+        PropertyMetadata $property,
+        $localeSource,
+        $localeDestination
+    ) {
+        if (!array_key_exists($property->getName(),
```

Comment:
@debreczeniandras sorry for a change request again. Did test it on my current webspace and we need to change also this lines to:

```
        if (!isset($structureArray[$property->getName()]['items'])) {
            return;
        }
```

as in some cases the property exist but could be null and isset would be enough for this case and error proof.

Reply:
@alexander-schranz no problem, it's now changed.

Reply:
@debreczeniandras  can you add the CHANGELOG.md entry also .